bindings/java/tools/build.py

```diff
@@ -75,8 +71,10 @@ def get_cargo_artifact_name(classifier: str) -> str:
     print('$ ' + subprocess.list2cmdline(cmd))
     subprocess.run(cmd, cwd=basedir, check=True)
 
+    # History reason of cargo profiles.
+    profile_output = 'debug' if args.profile in ['dev', 'test', 'bench'] else args.profile
```

Comment:
```suggestion
    profile = 'debug' if args.profile in ['dev', 'test', 'bench'] else args.profile
```

Keep name conventions.